LWA_EPIC/LWA_EPIC.py

```diff
@@ -2165,16 +2207,17 @@ def gen_args(return_parser=False):
         print("Output directory does not exist. Defaulting to current directory.")
         args.out_dir = "."
 
-    if args.removeautocorrs:
-        raise NotImplementedError(
-            "Removing autocorrelations is not yet properly implemented."
-        )
+    #if args.removeautocorrs:
+    #    raise NotImplementedError(
+    #        "Removing autocorrelations is not yet properly implemented."
+    #    )
```

Comment:
There have been some changes to the autocorr code, but has it been tested? As far as I know we don't run with this option, so I'm not sure this `NotImplementedError` should be removed yet.

Reply:
I'm not even sure if the auto-correlation removal ever worked.